### ISSUE TITLE
cannot use pm2 to start a python script that including pm2-python-interface

### ISSUE THREAD

**Issue by hccz95** [y4kupkaya/PM2#1]
I used `pm2.list()` in `test.py`, and then,
- if I execute `python test.py` in terminal, everything ok
- if I start `test.py` by `pm2 start test.json`, then got errors:
```
Traceback (most recent call last):
  File "/root/project/test.py", line 3, in <module>
    pm2.list()
  File "/root/miniconda3/lib/python3.12/site-packages/pm2/__init__.py", line 232, in list
    return [PM2Process(process) for process in loads(stdout)]
                                               ^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

- test.py
```python
from pm2 import PM2
pm2 = PM2()
pm2.list()
```

- test.json
```json
{
    "apps": [
        {
            "args": "",
            "name": "test",
            "script": "./test.py",
            "exec_interpreter": "/root/miniconda3/bin/python",
            "merge_logs": false,
            "watch": false,
            "error_file": "~/Pm2Logs/test/err.log",
            "out_file": "~/Pm2Logs/test/out.log"
        }
    ]
}
```

**Comment by y4kupkaya** [comment]
Fixed [[e053476](https://github.com/y4kupkaya/PM2/commit/e053476d0d105675a06a5d7f3afd0e861e73ac9a)] in the last update.

-> to update `pip install pm2 -U`

**_If you're still getting an error:_**

- [ ] Check if PM2 is installed on your device with the command `npm ls pm2 -g`.

_If your problem persists, you can open a new problem report on github without hesitation. Just give more details about the system and versions._

**New Pypi Version**: https://pypi.org/project/pm2/0.0.4/

**Comment by y4kupkaya** [comment]
**Fixed [[e053476](https://github.com/y4kupkaya/PM2/commit/e053476d0d105675a06a5d7f3afd0e861e73ac9a)] in the last update.**

-> to update `pip install pm2 -U`

**If you're still getting problems:**

- [ ] Check if PM2 is installed on your device with the command `npm ls pm2 -g`.

_If your problem persists, you can open a new problem report on github without hesitation. Just give more details about the system and versions._

**New Pypi Version:** https://pypi.org/project/pm2/0.0.4/

**Comment by y4kupkaya** [comment]
## 🚀 Major Update: PM2 Python Library v1.0.0 Released!

**Circular Dependency Issue Resolved!** The PM2 Python Library has been **completely rewritten from scratch** and is now available as **v1.0.0**! 🎉

### 🔄 Python Script + PM2 Integration:
This circular dependency issue has been **completely solved** with:
- **Clean architecture** - No more circular dependencies or conflicts
- **Isolated process management** - PM2 and Python scripts work harmoniously  
- **Proper subprocess handling** - Scripts can use PM2 library without conflicts
- **Production-grade design** - Used by enterprise applications safely

### ✨ What's New:
- **Complete rebuild** from the ground up
- **Zero legacy conflicts** - All circular dependency issues eliminated
- **Full PM2 API coverage** with clean separation of concerns
- **Async & Sync support** with proper resource management
- **Battle-tested architecture** - Production-ready reliability

### 📚 Resources:
- **📦 Install**: \pip install pm2\ (now on PyPI!)
- **🏠 Homepage**: https://projects.yakupkaya.me/pm2
- **📖 Documentation**: https://docs.yakupkaya.me/pm2
- **💻 Repository**: https://github.com/y4kupkaya/PM2

### �� Safe Python Script Example:
\\\python
# my_app.py - Can now safely use PM2 library
from pm2 import PM2Manager

def main():
    pm2 = PM2Manager()
    # This script can be started BY PM2 and still USE PM2
    other_processes = pm2.list_processes()
    print(f'Managing {len(other_processes)} processes')

if __name__ == '__main__':
    main()
\\\

\\\ash
# Start this script with PM2 - no more conflicts!
pm2 start my_app.py --name python-app
\\\

**The circular dependency issue is now completely resolved!** Please upgrade to v1.0.0 and test your use case. 🎯
